src/icons.scss

```diff
@@ -441,6 +441,11 @@
       @extend .fa-server;
       @extend .text-danger;
     }
+    &-busy {
```

Comment:
I called it 'busy' to match with the theme of the tooltip (which already existed).
is 'disabled' better ?